docker/Dockerfile.postgis

```diff
@@ -0,0 +1,24 @@
+FROM postgis/postgis:15-3.5
```

Comment:
It does not appear that this file needs to exist.  Remove and update compose file to use postgis directly